torch/csrc/jit/mobile/module.cpp

```diff
@@ -22,6 +22,15 @@ void CompilationUnit::register_function(std::unique_ptr<Function> fn) {
   methods_.emplace_back(std::move(fn));
 }
 
+Function* CompilationUnit::find_method_by_qn(const c10::QualifiedName& qn) {
```

Comment:
_by_qn feels redundant and they are Funcitons not methods, so maybe just `find_function`.